python/gradbench/gradbench/jax/gmm.py

```diff
@@ -0,0 +1,55 @@
+import jax
+import numpy as np
+
+jax.config.update("jax_enable_x64", True)
```

Comment:
I don't love mixing this line in among the imports; is there a better place to put it?

Reply:
Is it possible to scope it using a Python `with` statement instead?

Reply:
The documentation emphasizes this must be done "at startup", whatever that means. There is a context manager version, but it is marked as experimental. Perhaps we can get away with doing this later down, though. I'm mostly concerned with whether it has to occur before the @jit annotations.